app/Models/User.php

```diff
@@ -41,4 +41,7 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+    protected $guarded = [
+        "is_admin",
+    ];
```

Comment:
m ko the vua dung fillable va guarded voi nhau dc. Xem lai ly thuyet nha

Reply:
Em cảm ơn anh, em đã hiểu rồi ạ, em sẽ sửa lại và push lại code ạ.